app/assets/src/components/common/DetailsSidebar/PipelineStepDetailsMode/PipelineStepDetailsMode.jsx

```diff
@@ -77,13 +79,17 @@ PipelineStepDetailsMode.propTypes = {
     PropTypes.shape({
       fromStepName: PropTypes.string,
       files: PropTypes.arrayOf(
```

Comment:
You can define 
`
const FileList = PropTypes.arrayOf(
    PropTypes.shape({
      fileName: PropTypes.string.isRequired,
      url: PropTypes.string,
    }).isRequired
  )
`
and avoid the repetition.
If you use it other components, you can event extract it to an util file.